post-processing/post_process_nalu.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/python
+
+import numpy as np
+import os
+
+def parse_nalu_log(filename, equations, metric):
+    # Step 0: based on the metric chosen: avg, min or max
+    # set the correct offset when reading in data from log
+    offsets = dict({("avg", 3), ("min", 5), ("max", 7)})
+    offset  = offsets[metric]
+
+    # Step 1: read data from file
+    fi = open(filename, "r")
+    data = fi.readlines()
+    fi.close()
+
+    # Step 2: count total linear iterations per equation
+    numLinIters = np.zeros(len(equations), dtype=np.int32)
+    times       = np.zeros((len(equations), 8))
+    for lineIdx in range(len(data)):
+        line = data[lineIdx]
+	if("STKPERF: Total Time:" in line):
```

Comment:
Looks like wrong indentation.

Reply:
it's actually a tab vs whitespace issue, I will run a command that makes everything whitespaces.